### PR TITLE
History type for Gradio's Chat UI

### PR DESCRIPTION
**Pull request by tisnik**
## Description

History type for Gradio's Chat UI

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library
- [x] Trivial change

### CODE REVIEW

File: ols/src/ui/gradio_ui.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ def __init__(
     def chat_ui(
         self,
         prompt: str,
-        history,
+        history: list[list[Optional[str]]],
```
Comment (bparees) [inline ols/src/ui/gradio_ui.py:42]: this argument isn't even used, how did you conclude this should be its data type?

(I think we should just be deleting the argument)

Comment (tisnik) [reply ols/src/ui/gradio_ui.py:42]: is not `prompt` and `history` used automatically by the Chat widget?

Comment (tisnik) [reply ols/src/ui/gradio_ui.py:42]: and the conclusion - I looked into Gradio's sources https://github.com/gradio-app/gradio/blob/7302a6e151dac553c17833be64d4639ee4cf97aa/gradio/chat_interface.py

Comment (bparees) [reply ols/src/ui/gradio_ui.py:42]: sorry, you're correct:

from the gradio docs:

```
Should accept two parameters: a string input message and list of two-element lists of the form [[user_message, bot_message], ...] representing the chat history
```

**Pull request merged**